linera-views/src/dynamo_db.rs

```diff
@@ -1165,3 +1165,154 @@ impl IsResourceInUseException for SdkError<aws_sdk_dynamodb::error::CreateTableE
         )
     }
 }
+
+/// A static lock to prevent multiple tests from using the same LocalStack instance at the same
+/// time.
+static LOCALSTACK_GUARD: Mutex<()> = Mutex::const_new(());
```

Comment:
Here and in many places, we must really NOT compile test code in production, so please let's use `#[cfg(any(test, feature = "test"))]` and possibly group definitions into test-only modules.

Reply:
I added the `cfg` statements. But I prefer to put the DynamoDb code in a single file, just like for `ScyllaDb`, `RocksDb`, and `memory`.